ironmon_tracker/screens/InfoScreen.lua

```diff
@@ -910,7 +909,9 @@ function InfoScreen.drawAbilityInfoScreen(abilityId)
 		end
 	end
 
-	Drawing.drawButton(InfoScreen.Buttons.close, boxInfoBotShadow)
+	local closeButton = InfoScreen.Buttons.close
+	closeButton.boxColors = { "Upper box border", "Upper box background" }
+	Drawing.drawButton(InfoScreen.Buttons.close, boxInfoTopShadow)
```

Comment:
This permanently changes the color of the close button used for any info screen. The `closeButton` is still a reference to the original button.

A few solutions here are:
1. Temp store the colors, change them for the draw, then change back.
2. Use the bottom box color for the whole box, instead of top box color
3. Find a way to split the box into two, similar to Move Info Screen. We can likely use the top box for fun stats but I haven't thought about what those are.

I don't have a preference on which. I suppose at some point in the future we might end up on the third option.